cmd/snap/cmd_debug_recovery_key.go

```diff
@@ -0,0 +1,79 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !nosecboot
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/secboot"
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/release"
+)
+
+type cmdRecoveryKey struct{}
+
+func init() {
+	cmd := addDebugCommand("show-recovery-key",
+		"(internal) show the fde recovery key",
+		"(internal) show the fde recovery key",
+		func() flags.Commander {
+			return &cmdRecoveryKey{}
+		}, nil, nil)
+	cmd.hidden = true
+}
+
+func (x *cmdRecoveryKey) Execute(args []string) error {
+	if release.OnClassic {
```

Comment:
should we also limit to only run on uc20?

Reply:
I think we can. but given that it's a debug command and we get a somewhat useful error (that the recovery key file cannot be found) I think it's not worth it. But no super strong opinion here, just laziness :)